### PR TITLE
Use Github Actions instead of Travis

### PR DESCRIPTION
**Pull request by followyouhome**
### What was a problem?

All CI workflow was setup in Travis in sequence mode - PR check / release could took up to 30 minutes - nobody happy to wait 30 minutes to release his PR.

### How this PR fixes the problem?

* Use Github Actions as CI tool
* Split workflow into parallel jobs
* Use artifacts to prevent duplicated work between parallel jobs

### Check lists

- [ ] Readme file updated with actual description and example
- [ ] Added all ARIA attributes required for component
- [ ] Added tests for both browser and SSR render and for all components properties
- [ ] Added story with all knobs and actions

### Additional Comments (if any)

Yarn dependencies are not shared between parallel jobs, more info is presented in related ticket.

### CODE REVIEW

**Comment by adamaltman** [comment]
How long will it take now? I see "check snapshots" takes 11m.

**Comment by followyouhome** [comment]
About 15-17 mins.

File: .github/workflows/recomponents.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    branches: [master]
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+jobs:
+  compile:
+    name: Build dependencies
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
```
Comment (adamaltman) [inline .github/workflows/recomponents.yml]: Does it work with node 12.x?

Comment (followyouhome) [reply .github/workflows/recomponents.yml]: Checked, work with node 12.x

**Pull request merged**